src/parse.c

```diff
@@ -5572,6 +5572,9 @@ static type_t p_record_type_definition(ident_t id)
    // record element_declaration { element_declaration } end record
    //   [ simple_name ]
 
+   // VHDL-2019 allows for empty record declarations
+   // record { element_declaration } end record [ simple_name ]
+
```

Comment:
For the 2008 changes I've prefixed the grammar rules with "2008: ".

```suggestion
   // 2019: record { element_declaration } end record [ simple_name ]
```